### ISSUE TITLE
89115692 Add totals for Baptisms, Burials and Marriage in Decadal Display

### ISSUE THREAD

**Issue by FreeUKGenIssues** [FreeUKGen/FreeUKRegProductIssues#1018]
Priority 11 (0	1	3	7)

Issue reported by **STSCoord** at 2017-10-28 10:21:32 UTC
Time: 2017-10-28T10:18:04+00:00
Session ID: fb0307b326d67f0485f5fdc30c53d637
Problem Page URL: [/churches/5409d6aeeca9ebd28e29042b](/churches/5409d6aeeca9ebd28e29042b)
Previous Page URL: [https://www.freereg.org.uk/places/53cd4999eca9ebee4f00b979](https://www.freereg.org.uk/places/53cd4999eca9ebee4f00b979)
Reported Issue:
Would it be possible to put an entry at the bottom of the individual Baptisms, Marriages and Burials columns giving a total for that column?    

![Screenshot](http://www.freereg.org.uk/uploads/feedback/screenshot/59f45a2cf493fd0e1f1bf44d/_old_)

**Comment by richpomfret** [comment]
To ask for more info

**Comment by SteveBiggs** [comment]
Email sent requested justification.

**Comment by SteveBiggs** [comment]
Response from Michael Clarke:
"I have attached a sample of the display that I am talking about.

To my mind, the names in the top right box are redundant - each of the other column headers gives a list of transcribers/contributors so why repeat it?"



The "Total" column gives decadel coverage of all 10-year ranges giving a grand total of baptisms, burials and marriages for each range.

However, there is not a corresponding grand total of number of baptisms, etc. over all the decadels i.e. the answer to the question "How many baptisms are in this register?"

This is what I am looking for.

I must ask why there is actually any need for the "Total" column when it is irrelevant how many baptisms + burials + marriages there are per 10-year period but, much more relevant, is how many baptisms or burials or marriages there are."

![Capture.GIF](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/588631f51e19621a00230146/125516c66e82c728ace21e0d46cb958a379791accda5fd53b652fa912a5265296b51f02e7384e671e6162351580d4bac43585f45ffac8928f0a42062c80d55b98e30063e937f03a6aa8318b500bf21884882b2b98d4dc1ea3837aa.GIF)

**Comment by edickens** [comment]
I  agree with the points that Michael makes.  Perhaps just one grand total is needed at the top.
If there are to be totals of BA, MA and BU, I'd like to see them at the top of the columns.
Also, should we be showing email addresses?  However I appreciate that these are created by email addresses being in the Acknowledge field, but these be omitted if there is and @ in the data.  There are too many files to change.

**Comment by SteveBiggs** [comment]
Whilst I agree that the value of the right-hand overall totals column is questionable, I don't really see the value in having the total baptisms, etc in the register. A researcher wants to know the coverage for the period that their ancestor was born/married/buried, so they need the number of events during a given 10 year period. Knowing the overall total from 1538 to the present doesn't give them that information. Having said that, I'm not against putting in a total per column.

**Comment by PatReynolds** [comment]
No, we shouldn't be showing email address. GDPR may mean we need to remove all identification in public (including user ID).

**Comment by Sherlock21** [comment]
I support Michael's point and his request.

**Comment by Captainkirkdawson** [comment]
The grand total of al records is already there in the Register table all that is missing is the total by record type.

**Comment by Sherlock21** [comment]
Indeed it is, But its Totals by Type that would be useful as well please.

**Comment by stsccfr** [comment]
Closing as not valuable enough to implement
